README.md

```
@@ -64,6 +64,150 @@ Once the local Supabase setup is complete, you can develop with the full local s
+
+   - Manage Channels (to create channels)
+   - Send Messages (to post notifications)
+   - Read Message History (to read channel context)
+   - Mention Everyone (to @ mention users)
+   - Use External Emojis (optional, for better message formatting)
+   - Add Reactions (optional, for feedback on help requests)
+   - Manage Roles (to assign roles to users)
+
+   You can generate an invite URL with these permissions using the OAuth2 URL Generator in the Discord Developer Portal, or manually construct:
+
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)